src/probnum/diffeq/perturbedsolvers/_perturbation_functions.py

```diff
@@ -0,0 +1,58 @@
+"""perturbation functions to perturb the stepsize.
+
+References
+----------
+.. [1] https://arxiv.org/abs/1801.01340
+"""
+import numpy as np
+import scipy
+
+
+def perturb_uniform(step, solver_order, noise_scale, random_state=None, size=()):
+    """Perturb the step with uniformly distributed noise scaled by noise-scale [1]_.
+    proposed by Abdulle and Garegnani(2020)
```

Comment:
Can you use a line break before "proposed..." please? This is usual docstring style (see for instance the docstring style used by docformatter https://pypi.org/project/docformatter/).

Reply:
Sorry, super minor, but "Proposed by ..."?